other/files/000~ROOT~000-exposed.bcheck

```diff
@@ -0,0 +1,22 @@
+metadata:
+    language: v1-beta
+    name: "Filesystem exposure via /home/000~ROOT~000/"
+    description: "Tests for exposed 000~ROOT~000 in current path and at the root directory of site"
+    author: "r3nt0n"
+    tags: "exposure", "path traversal"
+
+run for each:
+  payloads_array = 
+    "/home/000~ROOT~000/",
+    `{base.request.url.path}/home/000~ROOT~000/`
+
+given request then
```

Comment:
For every request, you are trying this payload against the root as well as the current request path. This will result in a lot of repeated requests.

Does using `given path then` instead of `given request then` result in the behavior that you are looking for?